src/jdbc/cubrid/jdbc/jci/UServerSideConnection.java

```diff
@@ -1,6 +1,6 @@
 /*
- * Copyright (C) 2008 Search Solution Corporation
- * Copyright (C) 2016 CUBRID Corporation
+ * Copyright (C) 2008 Search Solution Corporation. 
```

Comment:
It is newly created in CUBRID.

Reply:
Thank you, I will remove a copyright of Search Solution at comment.